.gitmodules

```diff
@@ -119,3 +119,7 @@
 	url = https://github.com/nim-lang/checksums.git
 	ignore = untracked
 	branch = master
+[submodule "vendor/DOtherSide"]
```

Comment:
Are we back to a submodule?

Reply:
Yes, it was the easiest way for now. Will be removed totally in the future.

Reply:
In the submodule we have version combining history from old submodule and changes done in status-desktop directly. Next the history was purged, over 50 irrelevant commits removed. I don't want to spam `status-desktop` with all those changes. That's the rationale for doing it that way.

Reply:
Yeah, good! :+1: